manager.py

```diff
@@ -55,43 +71,121 @@ def configure(self, context):
         self.logger.info('REST Manager set to {}'.format(self._rest_manager))
 
         # fetch component settings
-        self._product_api_url_prefix = \
-            Settings.get("configuration", "product_api_url_prefix",
-                         fallback="https://api.nio.works/v1")
-
-        default = Persistence().load("api_key", default=None)
-        self._instance_api_key = \
-            NIOEnvironment.get_variable('API_KEY', default=default)
-
-        self._instance_id = Settings.get("configuration", "instance_id")
-       
+        self.config_api_url_prefix = \
+            Settings.get("configuration", "config_api_url_prefix",
+                fallback="https://api.nio.works/v1/instance_configurations")
+
+        setting_config_id = Settings.get("configuration", "config_id",
+            fallback=None)
+        self.config_id = Persistence().load("configuration_id",
+                                            default=setting_config_id)
+        setting_config_version_id = \
+            Settings.get("configuration", "config_version_id",
+                fallback=None)
+        self.config_version_id = Persistence().\
+            load("configuration_version_id", default=setting_config_version_id)
+        
         self._start_stop_services = Settings.getboolean(
             "configuration", "start_stop_services", fallback=True)
         self._delete_missing = Settings.getboolean(
             "configuration", "delete_missing", fallback=False)
 
-        # TODO in NIO-1141: Pull config settings for autonomy
+        # fetch instance specific settings
+        default = Persistence().load("api_key", default=None)
+        self.api_key = NIOEnvironment.get_variable('API_KEY', 
+                                                   default=default)
+
+        # config autonomy spcific settings
+        self._poll = Settings.getboolean("configuration", "config_polling",
```

Comment:
I believe we normally don't require a boolean setting for interval-like configurations, meaning a poll interval equal to 0 means no polling.